public/data/Deephunt3r.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Rakesh vaddi",
```

Comment:
Should `vadi` be capitalised to `Vaddi`? Thank you.

Reply:
@CodingSpecies Thanks for knowing me 👍🏻